config_converter/config_mapper/config_mapper.py

```diff
@@ -0,0 +1,198 @@
+"""Program that converts a fluentd config file to a master agent config file.
+
+Usage: Run the exec file in the bin folder with 2 arguments - the
```

Comment:
Btw, I see 3 arguments below.

Reply:
Yeah I forgot to update my documentation everywhere, but it should be accurate now. Thanks for pointing that out!